src/services/http_service.ts

```diff
@@ -83,6 +87,7 @@ export class HttpService {
       let fileParts = filePath.split(".");
       filePath = fileParts.pop();
 
+      // deno-lint-ignore no-explicit-any
       const database: any = Drash.Dictionaries.MimeDb;
```

Comment:
Again, isn't `unknown` a good fit for this?

Reply:
i defined an interface for the MIME DB dictionary: 3ac322e